scripts/build-dts.js

```diff
@@ -408,17 +408,116 @@
     exports.default = generate;
 });
 
-const prelude = `declare type bool = boolean;
-declare type i8 = number;
-declare type i16 = number;
-declare type i32 = number;
-declare type isize = number;
-declare type u8 = number;
-declare type u16 = number;
-declare type u32 = number;
-declare type usize = number;
-declare type f32 = number;
-declare type f64 = number;
+const prelude = `
+declare module "assemblyscript/std/assembly/shared/feature"{
```

Comment:
Wondering what's necessary to make the build step actually pick these up, instead of copy & pasting here.

Reply:
It's because for the sake of simplicity I extended "./node_modules/assemblyscript/std/portable.json".  I'll revert this change and just manually extend it.

Reply:
Would still prefer a better solution than copy & pasting the definitions below, which might change any time without anyone noticing.

Reply:
Hmm yeah in fact when this is used typescript doesn't like that `@unmaganged` isn't defined.  Is there a reason why these definitions need to be the same as those in `std`?

Do you have any ideas on how to solve this issue?  Currently I've given up on just depending on this directly and just depending on the parts of compiler that I need directly.  Especially because in my use cases I don't need binaryen.